apps/docs/src/app/core/component-docs/table/table-docs.component.html

```diff
@@ -164,6 +164,23 @@
 
 <separator></separator>
 
+<fd-docs-section-title [id]="'with-row-unnavigatable'" [componentName]="'table'">
+    Table with Non-navigatable Row
+</fd-docs-section-title>
+<description>
+    A row can be non-navigatable by setting the property <code>[navigatable]="false"</code>, for both regular and Pop In modes.<br><br>
+    When an entire row is used for navigation, the navigation indicator will hide when the row becomes non-navigatable.<br>
```

Comment:
Please rephrase this sentence. You can take a look at `Navigation indicators` for List component in Fund-styles. You can also refer to UX guidelines for more information.